### ISSUE TITLE
Discovery: Combined Profile and Account

### ISSUE THREAD

**Issue by joanneesteban** [department-of-veterans-affairs/va.gov-team#2329]
## User Story or Problem Statement
As an Authenticated Experience team member, I want to understand how to best measure the combined Profile and Account pages.

## Goal
- Separate KPIs by user intent

## Objectives or Key Results this is meant to further
- _lorem ipsum_

## Acceptance Criteria
- [ ] _What needs to happen or be created?_

## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `triage`, `tools-improvements`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `call center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

---

>>> Hi Joanne, Jon, and Nedie — 

Thanks for meeting with us yesterday to talk about our potential analytics approach for measuring success with the combine profile and account project. I'm creating this to-do so we can keep track of this conversation and regroup as needed.

These are my big takeaways from yesterdays call:

- We can potentially measure increased visits to the new profile/account as a metric for success, though our goal is to get the *right* people to this section more so than seeing an increase in the amount of unique visits. However, I think we can pretty safely say we don't want to see a reduced number of people to this new combined profile/account section from the amount of traffic we currently see to the separate profile and account pages.
- We would love to be able to measure reduced time to task completion for things like changing your address or updating your direct deposit information, but we don't have a way to measure time to task completion right now.
- We could potentially use an increase in people verifying their identity or adding 2FA as a metric for success since these things should be easier to find in this new section. However, it's worth noting we do prompt people to take these actions elsewhere, so we would have to measure this through increased link clicks from this section.
- We may be able to use exit rate to measure success, but it was not clear to me how we would do this.
- Reduction in profile-related calls to the call center could be a metric for success.

**To-dos**

- I think the only to-do from this meeting was that Nedie and Jon were going to take some time to think through this problem, and we'll regroup when there's more to discuss.

**Comment by jonwehausen** [comment]
CC: @Samara-Strauss 

Hi Team,

@nedierecel and I spoke today about tracking options for this request, here are some of the options we propose: 

**Main Ideas**
- Measure increase in key profile/account transactions, particularly profile form submissions and 2FA account upgrades. 
    - Rationale: _Because these items should now be easier to find, and we would see an increase in their usage_
- Track engagement time from new merged profile/account page to the first user action, classify what the action taken was: direct deposit, address update, etc. 
- Track time from when a user performs a search for 'direct deposit' or 'change address' to actually performing the action
  - Rationale: _It's pretty safe to say this was the user's intent if they searched these key words_
- Setup a 1-click foresee survey that displays after the user has updated their DD or address that says something like "You may have noticed our direct deposit / address has changed locations, how satisfied were you in finding this?" > We would then track user submissions of this question


**Supplemental Ideas**
- With an uncertainty of just how much the user would update this info, if an email went out about this update, we could track inbound clicks 
- We could setup a display banner that describes the move, and tracks user clicks on the banner

Let us know if you would like to discuss these options further. We look forward to hearing your thoughts.

**Comment by Samara-Strauss** [comment]
Thanks @jonwehausen and @nedierecel! Here are my thoughts:

> Measure increase in key profile/account transactions, particularly profile form submissions and 2FA account upgrades.

I think this makes sense. Re: 2FA, just be aware many people are defaulted into 2FA if they sign up through ID.me, so we would specifically want to look at transactions initiated from the profile.

> Track engagement time from new merged profile/account page to the first user action, classify what the action taken was: direct deposit, address update, etc.

So, to be clear, all of the profile features (contact, personal, military, and direct deposit info) are staying right where they are, and we are keeping the path to the profile exactly the same (Personalized Menu > Profile). The only content that is moving is what currently lives in Account Settings, and that's only a couple of things.

That being said, the new section should be easier to scan for information, so that *might* result in fewer people leaving because they can't find what they are looking for, and that *might* lead to decreased engagement time.

> Track time from when a user performs a search for 'direct deposit' or 'change address' to actually performing the action

This is a great idea. Idk if we do this now, but we should. 

However, I don't think that we would expect to see a decrease in the amount of time it takes someone to get to a feature through search, so this might not be a good comparison/before and after metric. When someone searches "direct deposit", the results will *not* send them to the profile — they are sent to a content page that routes them accordingly to either the profile or eBenefits depending on what they need to change direct deposit info for. This is how search works for all of our features — we direct to a content page instead of the tool because there are often multiple paths and multiple tools. 

TL;DR I don't think we'd expect to see a before/after change here between 1.0 and 2.0.

> Setup a 1-click foresee survey that displays after the user has updated their DD or address that says something like "You may have noticed our direct deposit / address has changed locations, how satisfied were you in finding this?"

The location of these items isn't changing, so this is a no-go. The only items that are changing are the account settings features, and honestly I don't think users would really care that they moved since this is rarely used functionality. 

Instead, maybe a Foresee survey with one question like "How easy was it for you to find what you were looking for in the Profile today?"

> With an uncertainty of just how much the user would update this info, if an email went out about this update, we could track inbound clicks

We don't send outbound emails about feature updates, so it's unlikely we could use this metric.

> We could setup a display banner that describes the move, and tracks user clicks on the banner.

We may do some sort of on-site banner announcing the feature update, and if we did, we'd definitely want to track clicks. But this doesn't really get at whether the feature update is an improvement or not.

-----

From everything we have discussed, these seem like the most helpful metrics:

- Measure visits to the new profile/account. The location of the profile isn't changing, so while we may not see an increase, we definitely shouldn't see a decrease.
- Measure increase in key profile/account transactions, particularly profile form submissions and 2FA account upgrades, since these things should be easier to find in the new, more scannable section. 
- Track engagement time from new merged profile/account page to the first user action, classify what the action taken was: direct deposit, address update, etc.
- Track time from when a user performs a search for 'direct deposit' or 'change address' to actually performing the action, though note we should not expect not see stark before/after metrics because of how search works.
- Setup a 1-click foresee survey that asks how easily people were able to find what they were looking for in the profile.
- We may be able to use exit rate to measure success, but it was not clear to me what this really tells us.
- Reduction in profile-related calls to the call center.

Qualitative metrics:

- These designs will be fully tested, and we can gauge participant's reaction to the 1.0 vs. 2.0 profile.

Any other thoughts? Happy to hop on another call to talk this out.

**Comment by nedierecel** [comment]
@Samara-Strauss Thanks for the feedback. We have a separate ticket that we are writing for implementation in #2330. 

Specific points:
1. Visits - Agreed. We should see it sustain or potentially increase, with the hypothesis that the more efficient combination of pages makes it easier for users to return.
2. Form Submissions and 2FA upgrades - Correct. These are already wired for Google Analytics; for the latter 2FA, we can separate that it occurred in the Profile vs. other tools that allow for upgrading (if they did not enroll with ID.me) 
3.  Time to first action - This is new implementation for us that will be covered in #2330. This is also our preferred method 
4. Time between search to action - As this is new, we will note any limitations in #2330. 
5. Foresee - We are working on Foresee customization in a separate story, but we will incorporate your feedback there. (CC: @joanneesteban for visibility)
6. Exit Rates - This is a secondary metric and not as useful as the aforementioned. Since the intent of the user is largely to check and quickly update data, my hypothesis is we would still see the same exit rates. 
7. Call Center Issues - Adding @clantosswett for visibility on this KPI

**Comment by Samara-Strauss** [comment]
@nedierecel thanks for this! All clear on my end. I'll add this to our product outline for tracking purposes.
